diff -uNr hashcat-5.1.0/src/affinity.c hashcat-5.1.0.mod/src/affinity.c
--- hashcat-5.1.0/src/affinity.c	2018-12-02 12:58:48.000000000 +0200
+++ hashcat-5.1.0.mod/src/affinity.c	2019-02-06 20:23:22.965321214 +0200
@@ -47,7 +47,7 @@
 
 int set_cpu_affinity (MAYBE_UNUSED hashcat_ctx_t *hashcat_ctx)
 {
-#if defined (__CYGWIN__)
+#if defined (__CYGWIN__) || defined(__ANDROID__)
   return 0;
 #else
   const user_options_t *user_options = hashcat_ctx->user_options;
